### ISSUE TITLE
Inconsistent constant expressions

### ISSUE THREAD

**Issue by Jesus89** [CartoDB/carto-vl#850]
There are some issues regarding constant expressions in Viz lang. Constant expressions can be constant numbers, palettes, images and placements. They are uppercase in the JS API and not sensitive in the String API. But there are some inconsistencies regarding names and namespaces.

### Constant Numbers

```js
carto.expressions.TRUE
carto.expressions.FALSE
carto.expressions.PI
...
```

```
true
False
PI
```

### Constant Palettes

There is no namespace in the String API.

```js
carto.expression.palettes.PRISM
carto.expression.palettes.EARTH
...
```

```
prism
earth
...
```

### Constant Images

There is no namespace in JS API or String API. Also, String API removes the underscore.

```js
carto.expressions.CAR
carto.expressions.TRIANGLE
carto.expressions.CIRCLE_OUTLINE
...
```

```
car
triangle
circleoutline
```

### Constant Placements

```js
carto.expressions.ALIGN_CENTER
carto.expressions.ALIGN_BOTTOM
...
```

```
align_center
align_bottom
```

My proposal is to review this from the point of view of the user and consider adding namespaces and more consistency to the API.

**Comment by Jesus89** [comment]
I agree with adding those namespaces. However, I feel that using `aligncenter` for the String API and `ALIGN_CENTER` could be confusing for the user because IMO you expect to have the underscore in both constants.

**Comment by elenatorro** [comment]
Yes, you are right. Although they are case agnostic, I would always show in the String API examples uppercase also to make it more intuitive.
